### DISCUSSION TITLE
Rotation Lag

### DISCUSSION THREAD

**Discussion by NiemandKG** [Sixze/ALS-Refactored#397] [Q&A]
How to add rotation lag for ALS_camera? 
![image](https://github.com/Sixze/ALS-Refactored/assets/74151950/27f81741-0806-4c0c-82c5-801821029d8c)

**Comment by ameaninglessname** [comment]
See `UAlsCameraConstants::RotationLagCurveName` and `UAlsCameraComponent::CalculateCameraRotation`, you could do that with `ALS` by adding a float animation curve named `RotationLag` to the character animations.
